optuna/distributions.py

```diff
@@ -144,7 +144,7 @@ def _contains(self, param_value_in_internal_repr):
         if self.low == self.high:
             return value == self.low
         else:
-            return self.low <= value and value < self.high
```

Comment:
It's not a strong opinion but how about creating another PR to fix code-style? (83f11e5 3943436 53ac93c)

Reply:
Thank you for your feedback. Sure

Reply:
Thank you for your feedback. Sure. I have created https://github.com/optuna/optuna/pull/1203